app/views/instrument_price_policies/_table.html.haml

```diff
@@ -0,0 +1,58 @@
+- clazz = price_policies.first.class
```

Comment:
it's a little odd to see `clazz` in a view, it makes me think there's about to be some metaprogramming or something going on

not really a big deal though